assets/js/modules/analytics-4/datastore/report.test.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * `modules/analytics-4` data store: report tests.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { MODULES_ANALYTICS_4 } from './constants';
+import {
+	createTestRegistry,
+	untilResolved,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import * as fixtures from './__fixtures__';
+
+describe( 'modules/analytics-4 report', () => {
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	describe( 'selectors', () => {
+		describe( 'getReport', () => {
+			const options = {
+				dateRange: 'last-90-days',
+				metrics: {
+					expression: 'testExpression',
+					alias: 'testAlias',
+				},
+			};
+
+			it( 'uses a resolver to make a network request', async () => {
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/analytics-4\/data\/report/,
+					{
+						body: fixtures.report,
```

Comment:
Ah ha. This is probably okay for now then, but we should update the fixture to be a valid GA4 report that would come from our API.